doc/changes/devel/12955.feature.rst

```diff
@@ -0,0 +1 @@
+Add :meth:`mne.io.base.BaseRAW.duration` property to centralize duration calculation for  :meth:`mne.io.base.BaseRAW.__repr__` and :meth:`mne.io.base.BaseRAW._repr_html_`, by :newcontrib:`Leonardo Rochael Almeida`.
```

Comment:
This file should be named `12955.newfeature.rst1` which is why towncrier is complaining.